corehq/apps/dump_reload/sql/dump.py

```diff
@@ -139,6 +139,12 @@
     FilteredModelIteratorBuilder('case_importer.CaseUploadFileMeta', SimpleFilter('caseuploadrecord__domain')),
     FilteredModelIteratorBuilder('case_importer.CaseUploadFormRecord', SimpleFilter('case_upload_record__domain')),
     FilteredModelIteratorBuilder('case_importer.CaseUploadRecord', SimpleFilter('domain')),
+    FilteredModelIteratorBuilder('motech.ConnectionSettings', SimpleFilter('domain')),
+    FilteredModelIteratorBuilder('repeaters.RepeaterStub', SimpleFilter('domain')),
+    # NH (2021-01-08): Including SQLRepeatRecord because we dump (Couch)
+    # RepeatRecord, but this does not seem like a good idea.
+    FilteredModelIteratorBuilder('repeaters.SQLRepeatRecord', SimpleFilter('domain')),
+    FilteredModelIteratorBuilder('repeaters.SQLRepeatRecordAttempt', SimpleFilter('repeat_record__domain')),
```

Comment:
For 99% of domains, including repeat records with dumped domain data is insignificant, but for a handful it could be gigantic, with very little benefit. I suspect that in the future, (SQL) repeat records will be deleted automatically after six to twelve months anyway.

I'd love to hear opinions on this, especially if you have dealt with domain migrations. (@esoergel @snopoke ?)

Reply:
We can certainly exclude all `SQLRepeatRecordAttempt` objects. Is it possible to only include `SQLRepeatRecord` objects that have not been successful yet?

Reply:
Good idea. Yeah, we can filter by `state`.